jenkins/ci.suse.de/cloud-socok8s.yaml

```diff
@@ -5,5 +5,13 @@
     # the credentialId for opensusegithubapi
     repo-credentials: c2350527-476a-45df-b406-84f028614682
     jobs:
-      - '{name}-integration'
```

Comment:
Are we getting rid of the current integration test?

Reply:
No, its just rename of existing integration test to make it clear that we are not running tempest test with PR change currently.